app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -43,187 +43,10 @@ def self.any_without_iso_datastores?
     without_iso_datastores.count > 0
   end
 
-  def supports_port?
-    true
-  end
-
-  def supported_auth_types
-    %w(default metrics)
-  end
-
   def supports_authentication?(authtype)
```

Comment:
@durandom wait no! thats not true, it will only add methods for the symbols in api3_supported_features and api4_supported_features, https://github.com/ManageIQ/manageiq/pull/11418/files#diff-d6125c21d43685840429020f8c5a57aaR186
other methods will not be effected. So if someone adds "authentication" symbol there, then it might cause a problem. But rightly so - if its not supposed to support authentication from some API version then it should be addressed.